browser/components/NoteItemSimple.js

```diff
@@ -30,6 +30,10 @@ const NoteItemSimple = ({ isActive, note, handleNoteClick, handleNoteContextMenu
         ? <i styleName='item-simple-title-icon' className='fa fa-fw fa-code' />
         : <i styleName='item-simple-title-icon' className='fa fa-fw fa-file-text-o' />
       }
+      {note.isPinned && !pathname.match(/\/home|\/starred|\/trash/)
```

Comment:
writing logic on stateless component does not make sense because testing component will be harder.
plz receive boolean of which means same as `!pathname.match(/\/home|\/starred|\/trash/`